src/main/kotlin/app/hashers/CodeLongevity.kt

```diff
@@ -209,7 +226,9 @@ class CodeLongevity(repoPath: String, tailRev: String) {
                         for (idx in delStart .. delEnd - 1) {
                             tmpLines.add(RevCommitLine(commit, oldPath, idx))
                         }
-                        lines.addAll(delStart, tmpLines)
+                        // TODO(alex): Approve code change.
+                        // delStart index caused IndexOutOfBoundException.
+                        lines.addAll(tmpLines)
```

Comment:
We really need unit tests, right @asurkov ?

Reply:
Absolutely. Btw, I always though that TODO(name) is about who added a todo item, not a person who's supposed to fix it.

Reply:
I think of this as a way to notify a person. Is there any conventions on that?